src/components/nav-search-button/nav-search-button.tsx

```diff
@@ -6,14 +6,16 @@ import {
   ElNavSearchButtonShortcutText,
 } from './styles'
 
-export interface NavSearchButton extends HTMLAttributes<HTMLButtonElement> {}
+export interface NavSearchButton extends HTMLAttributes<HTMLButtonElement> {
+  useShortcut?: boolean
```

Comment:
**issue (non-blocking):** `useShortcut` sounds like it will actually cause the shortcut to function... I think `isShortcutVisible` or `showHotKey` would be better (personally, I lean towards the terminology of "hot key" over "shortcut", but it's not a big deal).